api/KS.FiksProtokollValidator.WebAPI/Startup.cs

```diff
@@ -41,8 +53,9 @@ public AppSettings CreateAppSettings()
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
-            if (env.IsDevelopment())
+            if (env.IsDevelopment() || IsDockerCompose(env))
```

Comment:
Blir ikke docker-compose brukt under utvikling, eller tar jeg feil? Isåfall kunne dette kanskje vært et flagg :)

Reply:
IsDevelopment er lokalt på egen maskin og IsDockerCompose er ... Docker Compose. :) 
Måtte skille de to da det er forskjell på connection strings osv.

Reply:
Jeg trodde docker-compose bare var brukt lokalt, men mulig jeg misforstår isåfall  : )

Reply:
IsDevelopment er dotnet run fra terminal 👍